### PR TITLE
Fix pip and Git

### PR DESCRIPTION
**Pull request by Kwpolska**
1. Fix pip naming (PyPA is the organization behind pip)
2. Fix GIT → Git spelling (that’s the official way to spell it)
3. Install pip using `get-pip.py` instead of deprecated `easy_install`

Keep up the good work!

(PS. might be nice to clean up indentation and trailing whitespace in the script)

### CODE REVIEW

File: mac [hunk 1]
```diff
@@ -543,7 +543,7 @@ elif [ "$function" == "install" ]; then
 	echo "Installing pip..."
 	printf "${GREEN}sudo easy_install pip\n${NC}"
 	sudo easy_install pip
```
Comment (carhartl) [inline mac]: Shouldn't this be adapted as well?

**Comment by carhartl** [comment]
Another thing, when installing Python via brew, does it not come with pip out of the box?

File: mac (hunk 1 above)
Comment (Kwpolska) [inline mac]: Ah, the joys of code duplication. Fixed, thanks.

**Comment by carhartl** [comment]
http://stackoverflow.com/a/17271838

**Comment by Kwpolska** [comment]
I edited the code that was already there and assumed it was doing things correctly, but yes, brew should install pip. @guarinogabriel, perhaps `pip` installation should be removed completely and `brew install python` will take care of things.

**Comment by guarinogabriel** [comment]
@carhartl @Kwpolska – I agree with you. I will merge your PR and I will update the installation commands to use `brew install python` instead.

Thanks for your contribution!

**Comment by guarinogabriel** [comment]
@carhartl @Kwpolska – Actually, the `brew install python` command is already there. I will remove the pip installation from the install commands.

**Pull request merged**